src/orange/surf/Involute.cc

```diff
@@ -0,0 +1,32 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2023-2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file orange/surf/Involute.cc
+//---------------------------------------------------------------------------//
+#include "Involute.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+Involute Involute::at0origin(Real3 const& origin,
+                             real_type radius,
+                             real_type a,
+                             real_type sign,
+                             real_type tmin,
+                             real_type tmax)
+{
```

Comment:
I don't understand how this helper function is any different from the other? Did you mean to take `real_type z` and fill with `{0,0,z}`? I don't think that's worth the extra code.